pkg/controller/util/utils.go

```diff
@@ -52,3 +52,21 @@ func IsNodeAgentReady(c client.Client) bool {
 	}
 	return conf.Status.NodeAgent.Phase == csiv1.NodeAgentPhaseRunning
 }
+
+func Contains(list []string, s string) bool {
+	for _, v := range list {
+		if v == s {
+			return true
+		}
+	}
+	return false
+}
+
+func Remove(list []string, s string) []string {
+	for i, v := range list {
+		if v == s {
+			list = append(list[:i], list[i+1:]...)
```

Comment:
Does this function removes the first `s` or all of the occurrences of `s`?
If the answer is only the first occurrence, then we can break the loop here.
Otherwise, I'm not sure how Go iterates over a list while changing it.

Reply:
it removed only the first occurrence, but I improved it to remove all occurrences